appveyor.yml

```diff
@@ -1,17 +1,14 @@
-os: Visual Studio 2022
+os: Ubuntu2204
 
-before_build:
-  - nuget restore Renci.SshNet.sln
+services:
+- docker
 
-install:
-  - cinst dotnet-sdk --version=7.0.403 --limit-output
-
-build:
-  project: Renci.SshNet.sln
-  verbosity: minimal
+#for testing purposes only
+build_script:
+  - dotnet build test/Renci.SshNet.IntegrationTests/ -c Debug
 
 test_script:
-- cmd: >-
-    vstest.console /logger:Appveyor test\Renci.SshNet.Tests\bin\Debug\net462\Renci.SshNet.Tests.dll /TestCaseFilter:"TestCategory!=integration" --blame
-    
-    vstest.console /logger:Appveyor test\Renci.SshNet.Tests\bin\Debug\net7.0\Renci.SshNet.Tests.dll /TestCaseFilter:"TestCategory!=integration" --blame
+- mkdir artifacts
```

Comment:
```suggestion
- mkdir artifacts
- sudo tcpdump --list-interfaces
```

Reply:
1.eth0 [Up, Running, Connected]
2.any (Pseudo-device that captures on all interfaces) [Up, Running]
3.lo [Up, Running, Loopback]
4.docker0 [Up, Disconnected]
5.bluetooth-monitor (Bluetooth Linux Monitor) [Wireless]
6.nflog (Linux netfilter log (NFLOG) interface) [none]
7.nfqueue (Linux netfilter queue (NFQUEUE) interface) [none]
8.dbus-system (D-Bus system bus) [none]
9.dbus-session (D-Bus session bus) [none]